tests/issues.targets

```diff
@@ -261,6 +261,9 @@
         <ExcludeList Include="$(XunitTestBinBase)\JIT\Methodical\tailcall_v4\smallFrame\smallFrame.cmd">
              <Issue>tail. call pop ret is only supported on amd64</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)\JIT\opt\Tailcall\TailcallVerifyWithPrefix\TailcallVerifyWithPrefix.cmd" >
+            <Issue>x86 JIT doesn't support tail call opt</Issue>
```

Comment:
The reason TailcallVerifyWithPrefix and smallFrame fail on x86 is because the tests have `tail. call pop ret` sequence, which is only supported on amd64.